src/status.cc

```diff
@@ -14,6 +14,15 @@
 
 #include "status.h"
 
+#ifdef _WIN32
+#include "win32port.h"
```

Comment:
Is this actually necessary? As far as I know, MSVC with C++11 supports inttypes.h properly?